### PR TITLE
Update Smoke Testing advice

### LINKED ISSUE #33519: Smoke Testing updates: Release 6.3
https://github.com/mantidproject/documents/tree/main/Project-Management/Tools/RoadmapUpdate/SmokeTesting

- [x] Update the Smoke Testing create_issues.py script to work with conda, like the Manual test script
- [x] Also add a one second wait between post requests to prevent secondary rate limits: https://docs.github.com/en/rest/guides/best-practices-for-integrators#dealing-with-secondary-rate-limits
- [x] Consider how tests could be updated to narrow in on 1-1.5 hours max for SmokeTesting. (Ubuntu + Rhel7 too similar, after move to Conda?)

### PR DESCRIPTION
**Pull request by DanielMurphy22**
**Description of work.**

**To test:**

Fixes #33519 the points raised in the second comment: https://github.com/mantidproject/mantid/issues/33519#issuecomment-1033710455

Read the documentation updates and check that they clearly address the points raised in the issue.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by sf1919** [comment]
One thing I find extremely confusing about this page is the use of headings such as 'Monday, 3 weeks' and 'Thursday, 2 days'. What do they mean?

**Comment by DanielMurphy22** [comment]
> One thing I find extremely confusing about this page is the use of headings such as 'Monday, 3 weeks' and 'Thursday, 2 days'. What do they mean?

This never made sense to me but now I've added an update, I think it means time away from "the release day". So we Manual text with x weeks left and smoke test with 1 day left. I agree it's not clear. Not sure what would be better?

**Comment by sf1919** [comment]
Could we not just say '3 weeks before release due date' or something similar and remove the days of the week? I think a timeline is useful but it's the days that are causing the confusion.

File: dev-docs/source/ReleaseChecklist.rst [hunk 1]
```diff
@@ -110,21 +110,46 @@ with the Release Manager.
...
+* The QA manager should pre-setup 3 ISIS IDAaaS mantid dev instances and manually install the release package before testing
+  so the 1.5 hours is clear for testing time. Then share the instances with the relevant testers by providing their email in the workspace settings.
 
 Friday, 1 day
 -------------
 
+Smoke Testing
+^^^^^^^^^^^^^
+
+*  Make sure to follow the preparing steps listed above for yesterday.
```
Comment (sf1919) [inline dev-docs/source/ReleaseChecklist.rst:145]:
```suggestion
*  Make sure to follow the preparation steps listed above for yesterday.
```

**Comment by DanielMurphy22** [comment]
> Could we not just say '3 weeks before release due date' or something similar and remove the days of the week? I think a timeline is useful but it's the days that are causing the confusion.

@thomashampson @martyngigg (@sf1919) I thought I'd involve your opinions before making any changes. As the 4 of us are the ones involved in this page I think we should make it what we want! I'll admit I've used this page less than I probably should have in the past, but I agree with Sarah that at least initially these titles such as:
`Wednesday, 2 weeks & 3 days` are confusing. 

One options is to simply add `left` or `until release` to these titles. We could also remove the day of the week, unless you feel it adds value for your usual role? Open to suggestions!

**Comment by martyngigg** [comment]
I must admit to just being used to it and having done this too much that I don't actually look at the pages. The reorg by role was an improvement. Perhaps what would help would be defining a list of "events", such as feature freeze, start of beta testing etc and these are listed at the top. I'd say a diagram but they can be a pain to keep up to date. This way as the events may move in time the doc doesn't need to change - we just check locally what dates are set.

**Comment by sf1919** [comment]
I think having a list of events in order of when they need to occur is probably a better way forward and as @martyngigg says it's more flexible and less likely to need changing in the future.

File: dev-docs/source/ReleaseChecklist.rst [hunk 2]
```diff
@@ -12,131 +12,146 @@ request or perform a patch release look at the
...
+   are assigned tasks with the code expert in that testing team.
+
+Beta Testing Ends - Prepare for Smoke Testing
+#############################################
+
+* Liaise with the technical release manager and project manager to decide on an appropriate time for Smoke Testing.
+* Send an invite to developers for 1.5 hours maximum Smoke Testing. Include an introduction message to assign all testers to a certain operating system.
+  Link to the release pipeline builds where the release packages *WILL* be. Encourage testers to download
+  in the 30 mins before smoke testing. Inform that ticking on a testing issue means that someone has assigned themselves and will tackle that task.
+* The QA manager should pre-setup 3 ISIS IDAaaS mantid dev instances and manually install the release package before testing
```
Comment (sf1919) [inline dev-docs/source/ReleaseChecklist.rst:130]:
```suggestion
* The QA manager should pre-setup 3 ISIS IDAaaS Mantid dev instances and manually install the release package before testing
```

File: dev-docs/source/ReleaseChecklist.rst [hunk 3]
```diff
@@ -12,131 +12,146 @@ request or perform a patch release look at the
...
+   In short, testing teams of around 3-5 developers, spread across sub-teams
+   are assigned tasks with the code expert in that testing team.
+
+Beta Testing Ends - Prepare for Smoke Testing
+#############################################
+
+* Liaise with the technical release manager and project manager to decide on an appropriate time for Smoke Testing.
+* Send an invite to developers for 1.5 hours maximum Smoke Testing. Include an introduction message to assign all testers to a certain operating system.
+  Link to the release pipeline builds where the release packages *WILL* be. Encourage testers to download
+  in the 30 mins before smoke testing. Inform that ticking on a testing issue means that someone has assigned themselves and will tackle that task.
```
Comment (sf1919) [inline dev-docs/source/ReleaseChecklist.rst:129]:
```suggestion
  in the 30 minutes before smoke testing. Inform that ticking on a testing issue means that someone has assigned themselves and will tackle that task.
```

**Review by sf1919 — APPROVED** [review]
The revisions to the ReleaseChecklist make it so much easier to read. I'm so glad that the days of the week have been removed. I understand whats going on far better!

**Pull request merged**